guides/pre-release-week.md

```diff
@@ -56,7 +55,7 @@ Some tests still might fail on this main PR until the new release of MeiliSearch
 
 ## 🥳 After the MeiliSearch official release
 
-- Remove the `bump-meilisearch-v*.*.*-rc` branch from [meilisearch-digitalocean](https://github.com/meilisearch/meilisearch-digitalocean/branches).
+- Remove the `bump-meilisearch-v*.*.*-rc` branch and the `v*.*.*-rc` tag from the [cloud-scripts](https://github.com/meilisearch/cloud-scripts) repository.
```

Comment:
```suggestion
- Remove the `bump-meilisearch-vX.X.X-rc` branch and the `vX-X-X-rc` tag from the [cloud-scripts](https://github.com/meilisearch/cloud-scripts) repository.

```bash
# Delete local branch
$ git branch -d bump-meilisearch-vX.X.X-rc

# Delete remote branch
$ git push --delete origin bump-meilisearch-vX.X.X-rc

# Delete tag
$ git push --delete origin v.X.X.X-rc
```

```

Reply:
I would rather not add these explanations. This makes things complex and all these changes can be done via the GitHub interface.